src/Symfony/Bundle/TwigBundle/DependencyInjection/Configuration.php

```diff
@@ -127,7 +127,13 @@ private function addTwigOptions(ArrayNodeDefinition $rootNode)
                 ->scalarNode('cache')->defaultValue('%kernel.cache_dir%/twig')->end()
                 ->scalarNode('charset')->defaultValue('%kernel.charset%')->end()
                 ->booleanNode('debug')->defaultValue('%kernel.debug%')->end()
-                ->booleanNode('strict_variables')->end()
+                ->booleanNode('strict_variables')
+                    ->defaultValue(function () {
+                        @trigger_error('No setting "strict_variables" to false is deprecated as of Symfony 4.1, it will be the value of the "kernel.debug" parameter by default in 5.0.', E_USER_DEPRECATED);
```

Comment:
Not setting (missing "t")
this also misses some context IMHO.
Not setting the ... Twig configuration option ... ?

Reply:
Done, thanks!